chaoscenter/web/src/views/ExperimentBuilderTemplateSelection/blankCanvasTemplate.ts

```diff
@@ -64,7 +64,9 @@ function kubernetesBlankCanvasTemplate(
             name: '',
             image: `${imageRegistry.repo}/k8s:2.11.0`,
             command: ['sh', '-c'],
-            args: ['kubectl apply -f /tmp/ -n {{workflow.parameters.adminModeNamespace}} && sleep 30']
+            args: [
+              'kubectl apply -f /tmp/ -n {{workflow.parameters.adminModeNamespace}} && faultCount=$(ls /tmp/ | grep -E "(.yaml$|.yml$)" | wc -l) && until [[ $(kubectl get -f /tmp/ --no-headers | wc -l) -eq $faultCount ]]; do sleep 1; echo "Waiting for ChaosExperiment CR Ready..." ; done; echo "ChaosExperiment CR Ready"'
+            ]
```

Comment:
Hi @jemlog 
There are some edge cases which can happen here.
Lets say older CR for chaos experiments already exists on user's cluster. Now with this command, it will always pass & won't actually wait for new CR to get applied. Which is fine incase we were not making any changes to experiment CR. But with every release, we update go-runner in chaos-experiment CR. So, it's required that only when new experiment CR is applied on cluster, then only we should move on to other step.

Reply:
Hi @Jonsy13
To solve the edge case you mentioned, I included the process of deleting the existing ChaosExperiment CR first. 
I tried this solution because CR generation does not take much time. Please tell me if there are other edge cases. 
Thanks!